src/core/tsi/alts/handshaker/alts_tsi_handshaker.cc

```diff
@@ -38,10 +38,19 @@
 #include "src/core/tsi/alts/handshaker/alts_handshaker_client.h"
 #include "src/core/tsi/alts/handshaker/alts_shared_resource.h"
 #include "src/core/tsi/alts/zero_copy_frame_protector/alts_zero_copy_grpc_protector.h"
+#include "src/core/util/env.h"
 #include "src/core/util/memory.h"
 #include "src/core/util/sync.h"
 #include "upb/mem/arena.hpp"
 
+#define GRPC_EXPERIMENTAL_ALTS_HANDSHAKER_KEEPALIVE_PARAMS \
+  "grpc.alts_handshaker_keepalive_params"
```

Comment:
Please use the same value as in https://github.com/grpc/grpc-go/pull/8293.

Reply:
Done

Reply:
Same here, seems like this change did not get pushed?